src/cljs/rems/application.cljs

```diff
@@ -346,6 +349,10 @@
 (defn- application-licenses [application edit-application userid]
   (when-let [licenses (not-empty (:application/licenses application))]
     (let [application-id (:application/id application)
+          show-accepted-licenses?
+          (or (= (:application/applicant application) userid)
+              (contains? (set (map :userid (:application/members application)))
+                         userid))
```

Comment:
This could also be done, probably simpler, by checking whether `(:application/roles application)` contains `:applicant` or `:member`.

Reply:
Done and tested, seemed to work nicely..